### DISCUSSION TITLE
Monitoring - Install btop or glances

### DISCUSSION THREAD

**Discussion by bigheim** [amazonlinux/amazon-linux-2023#620] [General]
Is there anyway to install btop or glances on Amazon Linux 2023?

**Comment by bigheim** [comment]
Turns out I just downloaded the latest btop from git [here ](https://github.com/aristocratos/btop/releases/download/v1.3.0/btop-x86_64-linux-musl.tbz) and unzipped and the x86 binary in the bin directory worked just fine.

**Reply by danie-dejager** [reply]
You can create a repo for your al2023 hosts by using Fedora COPR to build packages for al2023. I've created a repo containing mostly interesting and missing packages by rebuilding Fedora or Centos 9 Stream packages. You can have a look here:
https://copr.fedorainfracloud.org/coprs/faramirza/al2023/
I don't add anything if it updates core AL2023 packages like systemd, openssh, openssl ect. I don't want an Amakenstein distro myself.

**Reply by sparrowt** [reply]
For the lazy, to just download it into current directory & run it:
```
wget https://github.com/aristocratos/btop/releases/download/v1.3.2/btop-x86_64-linux-musl.tbz
tar -xvf btop-x86_64-linux-musl.tbz
./btop/bin/btop
```

**Comment by colinbrislawn** [comment]
I'm running Graviton4 and Graviton5 vms on AWS.

Here's what works (fall 2026) after connecting to one of them with ssm:

```
cd #go to home dir that has write access

curl -LJO https://github.com/aristocratos/btop/releases/download/v1.4.7/btop-aarch64-unknown-linux-musl.tar.gz
tar -xvf  btop-aarch64-unknown-linux-musl.tar.gz
./btop/bin/btop

alias btop="/home/ssm-user/btop/bin/btop"
```
